src/common/cockpitpipe.c

```diff
@@ -1407,7 +1408,7 @@ cockpit_pipe_pty (const gchar **argv,
   pid = forkpty (&fd, NULL, NULL, &winsz);
   if (pid == 0)
     {
-      if (cockpit_unix_fd_close_all (3, -1) < 0)
+      if (cockpit_close_range (3, G_MAXINT, 0) < 0)
```

Comment:
It's a bit weird to see the glib macro used with a "pure C" call.  Maybe that's because you'd need another header for the libc one?

Reply:
Replaced with `INT_MAX`.